numba/cuda/tests/cudapy/test_inspect.py

```diff
@@ -55,23 +55,23 @@ def foo(x, y):
         # Signature in LLVM dict
         llvmirs = foo.inspect_llvm()
         self.assertEqual(2, len(llvmirs), )
-        self.assertIn((self.cc, (intp, intp)), llvmirs)
-        self.assertIn((self.cc, (float64, float64)), llvmirs)
+        self.assertIn((intp, intp), llvmirs)
```

Comment:
Should the removal of `cc` be noted somewhere? Docs? IIRC OOB we discussed a deprecation notice to be added.

Reply:
Further to OOB conversation, I've added back support for `(cc, argtypes)` indexing (and tests below) as well as a deprecation notice in https://github.com/numba/numba/pull/6432/commits/bd4a188bcee7be2e1aafd6d2d05a249039315243